WebRoot/js/zimbraMail/briefcase/view/ZmPreviewPaneView.js

```diff
@@ -385,6 +386,19 @@ ZmPreviewPaneView.prototype._listSelectionListener = function(ev, item) {
     	} else {
     		this._previewView.set(item);
     	}
+    } else if (!this._controller.isReadingPaneOn() && item && item.folderId == ZmFolder.ID_FILE_SHARED_WITH_ME) {
+        var restUrl = item.getRestUrl();
+        restUrl = AjxStringUtil.fixCrossDomainReference(restUrl);
+        var req = new XMLHttpRequest();
+        req.open("GET", restUrl, false);
```

Comment:
why third parameter as false?

Reply:
this is to make request synchronous.

Reply:
we don't use synchronous requests in classic ui

Reply:
converted it to async request